Makefile

```diff
@@ -95,16 +95,17 @@ $(NODE_CONTAINER_BIN_DIR)/confd:
 
 # Get the calico-bgp-daemon binary
 $(NODE_CONTAINER_BIN_DIR)/calico-bgp-daemon:
+	$(CURL) -L $(GOBGP_URL) -o $(@D)/gobgp
```

Comment:
I would suggest leaving in now, and possibly removing if you get gobgp status working.   WDYT?